extensions/ql-vscode/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # CodeQL for Visual Studio Code: Changelog
 
-## [UNRELEASED]
+## 1.6.12 - 1 September 2022
 
 - Add ability for users to download databases directly from GitHub. [#1485](https://github.com/github/vscode-codeql/pull/1485)
+- Fix a race condition that could cause a failure to open the evaluator log when running a query. [#1490](https://github.com/github/vscode-codeql/pull/1490)
+- Fix an error when running a query with an older version of the CodeQL CLI. [#1490](https://github.com/github/vscode-codeql/pull/1490)
```

Comment:
Both fixed in the same PR. That's fine. Just interesting seeing them one after another.

Reply:
Yeah, probably should have split them up for the same of the change log.